### ISSUE TITLE
multi-organ segmentation supporting

### ISSUE THREAD

**Issue by shiyu2011** [twni2016/OrganSegRSTN_PyTorch#8]
Is this course-fine model supporting multi-organ segmentation simultaneously?

**Comment by twni2016** [comment]
Sorry, not.

**Comment by shiyu2011** [comment]
Thanks @twni2016 for sharing this good work. To support multi-organ segmentation simultaneously, could I apply saliency transformation over each organ, crop it and pass it through fine_model and return the probability map belongs to each class for joint optimization? please advise.

**Comment by twni2016** [comment]
Yes, you may try this idea to implement multi-organ segmentation. I would suggest a global coarse seg model to output the prob maps of multiple organs, and crop each channel and feed them into the corresponding fine seg models.
